app/parse.py

```diff
@@ -8,8 +14,55 @@ class Quote:
     tags: list[str]
 
 
+def parse_single_quote(quote_soup: BeautifulSoup) -> Quote:
+    text = quote_soup.select_one(".text").get_text(strip=True)
+    author = quote_soup.select_one(".author").get_text(strip=True)
+    tags = [tag.get_text(strip=True) for tag in quote_soup.select(".tag")]
+
+    return Quote(text=text, author=author, tags=tags)
+
+
+def parse_quotes_page() -> [Quote]:
```

Comment:
The parse_quotes_page function name implies that it parses a single page, but it actually parses all pages. A better name might be parse_all_quotes.